openstack/nova/templates/db-migrate-job.yaml

```diff
@@ -37,50 +37,41 @@ spec:
           value: {{ .Values.python_warnings | quote }}
         volumeMounts:
         - mountPath: /etc/nova
-          name: etcnova
-        - mountPath: /etc/nova/nova.conf
           name: nova-etc
-          subPath: nova.conf
-          readOnly: true
-        - mountPath: /etc/nova/nova-api.conf
-          name: nova-etc
-          subPath: nova-api.conf
-          readOnly: true
-        - mountPath: /etc/nova/api-paste.ini
-          name: nova-etc
-          subPath: api-paste.ini
-          readOnly: true
```

Comment:
Any idea why we would need api-config and policy for the migration job?

Reply:
No idea. I suspect it is a simply the result of copy-and-paste, but I didn't want delay my upgrade of kvm to xena to test which of the files are neccessary.